model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -1,9 +1,23 @@
 
 def format_import_proto_library(name, src, deps):
-  """
-  Formats the import statement of the src file to contain a full path to the file
-  The genrule() rule generates a copy of the src file and alters the copy
-  The proto_library() rule takes this altered copy and builds it
+  """This macro exists as a way to build proto files that contain import statements in both Gradle
+  and Bazel.
+  Currently the files that use this macro are: exploration.proto, topic.proto, and question.proto.
```

Comment:
Let’s not relist the uses here since this can quickly become out of date. Devs can search for the uses if they want to know them.

Reply:
Ok! Makes sense.